transformers/storage/cat/repository.go

```diff
@@ -51,23 +51,38 @@ func (repository *StorageRepository) SetDB(db *postgres.DB) {
 }
 
 func (repository *StorageRepository) insertLive(diffID, headerID int64, live string) error {
-	_, err := repository.db.Exec(insertCatLiveQuery, diffID, headerID, live)
+	addressID, addressErr := shared.GetOrCreateAddress(repository.ContractAddress, repository.db)
+	if addressErr != nil {
+		return fmt.Errorf("Could not retrieve address id for %s, error: %w", repository.ContractAddress, addressErr)
```

Comment:
incredibly minor but I think the linter complains about capitalization in error strings

Reply:
Oh bite me linter. why can no haz capitals?